src/main/java/sg/ncl/testbed_interface/Team2.java

```diff
@@ -1,14 +1,19 @@
 package sg.ncl.testbed_interface;
 
+import lombok.Getter;
+import sg.ncl.domain.MemberStatus;
+
 import javax.validation.constraints.Size;
 import java.io.Serializable;
+import java.lang.reflect.Member;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 
 /**
  * @author Te Ye
  */
+@Getter
 public class Team2 implements Serializable {
 
     private String id;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "id" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)